hbase-server/src/main/java/org/apache/hadoop/hbase/master/DeadServer.java

```diff
@@ -96,29 +75,6 @@ synchronized boolean areDeadServersInProgress() {
    */
   synchronized void putIfAbsent(ServerName sn) {
     this.deadServers.putIfAbsent(sn, EnvironmentEdgeManager.currentTime());
-    processing(sn);
-  }
```

Comment:
These codes are not only set the server be dead, but also set it be processing.
But the changed codes only set it be dead.
As a result ,when calling `ServerManager.expireServer()` method, the server is firstly set be dead by `moveFromOnlineToDeadServers()`, but the SCP is submitted afterwards. 
So this patch made a diff from the origin codes when calling `ServerManager.areDeadServersInProgress()`, because the new codes has a little delay time of submit and execute the SCP procedure.